clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -11524,6 +11524,8 @@ def err_sycl_invalid_accessor_property_list_template_param : Error<
   "%select{parameter pack|type|non-negative integer}1">;
 def err_sycl_num_kernel_wrong_reqd_wg_size : Error<
   "%0 attribute must evenly divide the work-group size for the %1 attribute">;
+def err_sycl_invalid_aspect_argument : Error<
+  "invalid argument for %0 attribute; argument must be device aspect of type cl::sycl::aspect">;
```

Comment:
Can we update diagnostic message like -
def err_sycl_invalid_aspect_argument : Error<
  "%0 attribute argument is invalid; must be device aspect of type cl::sycl::aspect">;

Reply:
Kind of a nit, but we are trying to encourage people to use the `sycl::` namespace rather than `cl::sycl::`.  Can we change the wording to say `... of type sycl::aspect` instead?